Packet++/src/TcpReassembly.cpp

```diff
@@ -41,6 +42,8 @@ TcpReassembly::TcpReassembly(OnTcpMessageReady onMessageReadyCallback, void* use
 	m_ClosedConnectionDelay = (config.closedConnectionDelay > 0) ? config.closedConnectionDelay : 5;
 	m_RemoveConnInfo = config.removeConnInfo;
 	m_MaxNumToClean = (config.removeConnInfo == true && config.maxNumToClean == 0) ? 30 : config.maxNumToClean;
+	// if m_MaxOutOfOrderFragments is set to numeric_limits<size_t>::max() there is no limit, since size() cannot return a value bigger than that
+	m_MaxOutOfOrderFragments = (config.maxOutOfOrderFragments ? config.maxOutOfOrderFragments : std::numeric_limits<size_t>::max());

```

Comment:
let's not use `<limits>` and `std::numeric_limits<size_t>::max()` because I don't think it's supported on all platforms. Instead, we have a few options:
- Set the max possible value to some very high number, let's say `0xffffffff`
- Convert `m_MaxOutOfOrderFragments ` to `int` and set its default value to `-1` which means "unlimited". Then add a check whether this number is negative and treat it as unlimited